.changeset/pink-roses-confess.md

```diff
@@ -0,0 +1,12 @@
+---
+"@logto/experience-legacy": patch
+"@logto/experience": patch
+---
+
+fix(experience): properly filter WeChat connectors by platform (Web | Native) in SSR sign-in experience settings
```

Comment:
@charIeszhao should follow our convention?

Reply:
fixed